### PR TITLE
Updated draft of Trevor's glibc vs musl draft

### PR DESCRIPTION
**Pull request by SharpRake**
I was attempting to make changes to @tdunlap607's glibc vs musl draft (PR is here: https://github.com/chainguard-dev/edu/pull/1788) but I ran into problems when I tried to update his PR. This is a fresh PR with my updates on his draft.

I wouldn't say it needs a close review, as this drat is the result of my own review.

you can view the draft here: https://deploy-preview-1799--ornate-narwhal-088216.netlify.app/chainguard/chainguard-images/working-with-images/compiled-programs/glibc-vs-musl/

### CODE REVIEW

File: content/chainguard/chainguard-images/working-with-images/compiled-programs/glibc-vs-musl.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+| ------------------- | -------------------------------------------------------------- | ---------------------------------------------------------------------- |
+| First Release   	| 1988                                                       	| 2011                                                               	|
+| License         	| GNU Lesser General Public License (LGPL)                   	| MIT License (more permissive)                                      	|
+| Binary Size     	| Larger Binaries                                            	| Smaller Binaries                                                   	|
+| Runtime Performance | Optimized for performance                                  	| Slower performance                                                 	|
+| Build Performance   | Slower                                                     	| Faster                                                             	|
+| POSIX Compliance	| High, complete implementation                              	| Subset, core functionalities                                       	|
```
Comment (tdunlap607) [inline content/chainguard/chainguard-images/working-with-images/compiled-programs/glibc-vs-musl.md:41]: I apologize, but can we delete the POSIX Compliance row and change to this:

| Compatibility	| POSIX Compliant + GNU Extensions                              	| POSIX Compliant

Comment (SharpRake) [reply content/chainguard/chainguard-images/working-with-images/compiled-programs/glibc-vs-musl.md:41]: on it!

File: content/chainguard/chainguard-images/working-with-images/compiled-programs/glibc-vs-musl.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+Additionally, Node.js mentions in their [building documentation](https://github.com/nodejs/node/blob/main/BUILDING.md#platform-list): "For production applications, run Node.js on supported platforms only." musl and Alpine are experimental supports, whereas glibc is Tier 1 support.
+
+The [Go](https://hub.docker.com/_/golang) image also mentions that Alpine is not officially supported and experimental: "This (Alpine) variant is highly experimental, and *not* officially supported by the Go project (see [golang/go#19938⁠](https://github.com/golang/go/issues/19938) for details)."
+
+## Unsupported Debug Features
+
+Certain applications that rely on debug features for testing — including [sanitizers](https://wiki.musl-libc.org/open-issues#Sanitizer-compatibility) (including [Addressanitizer](https://clang.llvm.org/docs/AddressSanitizer.html), [threadsanitizer](https://clang.llvm.org/docs/ThreadSanitizer.html), etc.) and profilers (such as [gprof](https://ftp.gnu.org/old-gnu/Manuals/gprof-2.9.1/html_mono/gprof.html)) — are not supported by musl.
```
Comment (erikaheidi) [inline content/chainguard/chainguard-images/working-with-images/compiled-programs/glibc-vs-musl.md:285]:
```suggestion
Certain applications that rely on debug features for testing — including [sanitizers](https://wiki.musl-libc.org/open-issues#Sanitizer-compatibility) (such as  [Addressanitizer](https://clang.llvm.org/docs/AddressSanitizer.html), [threadsanitizer](https://clang.llvm.org/docs/ThreadSanitizer.html), etc.) and profilers (such as [gprof](https://ftp.gnu.org/old-gnu/Manuals/gprof-2.9.1/html_mono/gprof.html)) — are not supported by musl.
```

**Review by erikaheidi — APPROVED** [review]
Looks good, I just added a few small suggestions. Great piece of content, I learned a lot from it!

File: content/chainguard/chainguard-images/working-with-images/compiled-programs/glibc-vs-musl.md (hunk 2 above)
Comment (SharpRake) [reply content/chainguard/chainguard-images/working-with-images/compiled-programs/glibc-vs-musl.md:285]: ooh, good catch!

**Pull request merged**